### ISSUE TITLE
user.mod missing

### ISSUE THREAD

**Issue by Mittagskogel** [ExtremeFLOW/neko#1140]
If I follow the instructions to run the `tgv` example, makeneko fails due to a missing `user.mod` module. I cannot see the mod file in the include directory and I am unable to find anything relevant in the documentation. Where should the user.mod file be coming from?

**Comment by Mittagskogel** [comment]
Yep, I can confirm that this fails in develop. I think the reason is that there are some undefined types, which causes errors and prevents user.mod from being built.

**Comment by timfelle** [comment]
Interesting, it does not fail on my computer. Try the following:
- `git clean -fdx`, to clean the neko directory avoiding compilation not refreshing old files.
- Recompile neko
- cd to the tgv example directory.
- `makeneko tgv.f90`

Could you try this and confirm that you get something similar to the message bellow.

```
N E K O build tool, Version 0.7.99
(build: 2024-02-14 on x86_64-pc-linux-gnu using gnu)

Building user NEKO ... done!
```

**Comment by Mittagskogel** [comment]
Yes, that works, so something must've been corrupted on my side. Thank you!
